cif/products/src/mappers/PagedResponseProductMapper.ts

```diff
@@ -112,4 +118,25 @@ export default class PagedResponseProductMapper extends Mapper<PagedResponseProd
       totalResults,
     });
   }
+
+  private buildCIFRequest(mappable: any) {
+
+    const {
+      offset = 0,
+      limit = 20,
+      filter = '',
+    } = mappable;
+
+    const cifRequestType = CommerceIntegrationHelper.checkRequestType(filter);
+    switch (cifRequestType) {
+      case CIFRequestType.Variant: {
+        const selectedVariant = filter.match(/(?<=")[^"]+(?=")/);
+        return { query: `::code:${selectedVariant}`, pageSize: limit, currentPage: offset / limit };
```

Comment:
simply return  `::code:${selectedVariant}` and put this value inside the query on mapFromInputArgsToActionParameters

Reply:
done